src/utils/map-controls.js

```diff
@@ -108,10 +108,10 @@ export default class MapControls {
     const oldViewport = this.mapState.getViewportProps();
     const newViewport = newMapState.getViewportProps();
 
-    if (this.onChangeViewport &&
+    if (this.onViewportChange &&
```

Comment:
This is the same prop. The MapControls class is the one firing the callback in the current design. Idea being a user can add their own callbacks (onMapMove, onMapRotate for example) by supplying their own control class.